### ISSUE TITLE
NoSuchMethodError: The method '[]' was called on null.

### ISSUE THREAD

**Issue by dougti** [invoiceninja/flutter-mobile#48]
Hi,
 When I open the app & land on the Overview tab of the Dashboard, I just get the errors displayed in the below images. The screen in the second image is displayed when I try to open the menu.

<img src="https://user-images.githubusercontent.com/13472166/51009872-9ee60900-1530-11e9-8b94-93a142ac1a9b.png" width="300px" style="display:inline;">

<img src="https://user-images.githubusercontent.com/13472166/51009873-9ee60900-1530-11e9-9071-90c68a755b29.png" width="300px"  style="display:inline;">

Everything accessed via the Activity tab appears to be working as expected.

Invoice Ninja 4.5.8
PHP-FPM 7.2
Latest Android app

**Comment by dougti** [comment]
> Are you using a language supported by the app?

I suspected that may be the issue, however non of the docs mention what languages are or are not supported on the app.

Although strings with missing translations usually fall back to the default English, no? I've noticed there are already several strings doing exactly that in the areas of the app that _do_ work.
For example: "Overview", "Total Amount", "Invoice Status", "Past Due".

As only one global language is supported for all users & clients, my site's configured to use Brazilian Portuguese.

I'd be happy to contribute to the translations. Does the app use the same Transifex project as the web app? I've just sent a request to join the team.
